File: python/509_斐波那契数_easy.py
class Solution:
    def fib(self, n: int) -> int:
        if n == 0:
            return 0
        elif n == 1:
            return 1
        f0, f1 = 0, 1
        for i in range(2, n+1):
            f0, f1 = f1, f0+f1
        return f1

File: cpp/509_斐波那契数_easy.cpp
class Solution {
public:
    int fib(int n) {
        if(n==0)
            return 0;
        else if(n==1)
            return 1;
        int f0 = 0, f1 = 1;
        for(int i=2; i<=n; i++){
            int tmp = f1;
            f1 = f0 + f1;
            f0 = tmp;
        }
        return f1;
    }
};